app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -53,6 +53,34 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
           N_('Stop Server'),
           :image   => "guest_shutdown",
           :confirm => N_("Do you want to stop this server?")),
+        button(
+          :middleware_deployment_add,
+          'pficon pficon-add-circle-o fa-lg',
+          N_('Add a new #{ui_lookup(:table=>"middleware_deployment")}'),
+          N_('Add Deployment'),
+          :data => {'toggle'        => 'modal',
+                    'target'        => '#modal_d_div',
+                    'function'      => 'miqCallAngular',
+                    'function-data' => '{"name": "showListener", "args": []}'})
+      ]
+    ),
+  ])
+  button_group('middleware_server_deployments', [
+    select(
+      :middleware_server_deployments_choice,
+      'pficon pficon-save fa-lg',
+      t = N_('Deployments'),
+      t,
+      :items => [
+        button(
+          :middleware_deployment_add,
```

Comment:
Yes, removed the button that was inside the "Power" dropdown and just left a single dropdown for "Deployments".